service/src/main/java/com/ge/predix/acs/attribute/readers/AttributeRetrievalException.java

```diff
@@ -8,7 +8,15 @@ public AttributeRetrievalException(final String message, final Throwable cause)
         super(message, cause);
     }
 
+    public AttributeRetrievalException(final String message) {
+        super(message);
+    }
+
     static String getAdapterErrorMessage(final String adapterEndpoint) {
         return MessageFormat.format("Couldn''t get attributes from the adapter with endpoint ''{0}''", adapterEndpoint);
     }
+
+    static String getStorageErrorMessage(final String id) {
+        return String.format("Total size of attributes or number of attributes to large for id: '%s'", id);
```

Comment:
Typo: to -> too.

Also is it worth mentioning the thresholds in the message at the time this exception is thrown to clue the user in (in case something can be done to proactively limit the attributes on the adapter side)?